sdk/file_sdk/Common/file_format.h

```diff
@@ -0,0 +1,34 @@
+#ifndef FILE_FORMAT_H
+#define FILE_FORMAT_H
+
+#include "ifile_format.h"
+
+namespace mip {
+namespace file {
+
+class FileFormat : public IFileFormat
+{
+public:
+  FileFormat(std::shared_ptr<IStream> file,

```

Comment:
this is an "abstract" class

---
In reply to: [134446626](https://github.com/AzureAD/rms-sdk-for-cpp/pull/133#discussion_r134446626) [](ancestors = 134446626)